sympy/core/power.py

```diff
@@ -822,6 +822,34 @@ def _eval_is_composite(self):
     def _eval_is_polar(self):
         return self.base.is_polar
 
+    def combines(self, x):
+        """return +/-1 if ``(b**e)**x == +/-b**(e*x)`` else None"""
+        try:
+            as_int(x, strict=False)
+            return S.One
+        except ValueError:
+            b, e = self.as_base_exp()
+            p = Pow._eval_power(
+                Pow(b, e, evaluate=False), x)
```

Comment:
In #22052 I deliberately changed this so that it did not depend on `_eval_power` but rather a principled check based on the available assumptions. Is it not possible to do the same here?

Reply:
`_eval_power` *is* the principled check of conditions. It was my goal (long ago) to remove all those decisions from everywhere in the code base and localize them there.

I experimented with pulling out the code from there and putting it in `combines` and then having `_eval_power` ask `combines` if the exponents should be joined, but the results of `_eval_power` are a bit more general than the notion of `combines` so it didn't look that clean. So I decided to make `combines` query the result of `_eval_power`.